plugins/inputs/ping/ping.go

```diff
@@ -14,7 +14,7 @@ import (
 	"sync"
 	"time"
 
-	"github.com/go-ping/ping"
+	probing "github.com/prometheus-community/pro-bing"
```

Comment:
Let's keep the import name ping. That will cut down on unnecessary changes in the PR

```suggestion
	ping "github.com/prometheus-community/pro-bing"
```